### PR TITLE
changing tests/README.md to reflect repo structure change

### PR DESCRIPTION
**Pull request by nitinramvelraj**
## What does this PR do?

Changing tests/README.md to reflect changes in repo structure.

### Does your PR introduce any breaking changes? If yes, please list them.

No

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ To setup a local development environment, install both local and test dependenci
 
 ```bash
 # clone the repo
-git clone https://github.com/PyTorchLightning/pytorch-lightning
-cd pytorch-lightning
+git clone https://github.com/Lightning-AI/lightning.git
+cd lightning
 
 # install required depedencies
 python -m pip install ".[dev, examples]"
```
Comment (carmocca) [inline tests/README.md:16]: @Borda these need to be updated too, right?

Comment (Borda) [reply tests/README.md:16]: well yes, and atm we do not have joined extras if you install all, so if you develop Pl you shall run (and I would emphasise to install in developer mode `-e`)
```
export PACKAGE_NAME=pytorch
pip install -e ".[dev, examples]"
```
https://github.com/Lightning-AI/lightning/blob/c71f32a490dcbfea20882cf2e9d8b1439c492142/setup.py#L27

**Review by Borda — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]
It looks good to me 🚀

**Comment by nitinramvelraj** [comment]
Hi, is there anything left to be done on this PR?

File: tests/README.md (hunk 1 above)
Comment (rohitgr7) [reply tests/README.md:16]: @Borda

Comment (Borda) [reply tests/README.md:16]:
```suggestion
export PACKAGE_NAME=pytorch
python -m pip install ".[dev, examples]"
```

**Pull request merged**